docs/recipes/install/common/add_ww4_hosts_finalize.tex

```diff
@@ -6,7 +6,7 @@
 \begin{lstlisting}[language=bash,keywords={},upquote=true,basicstyle=\footnotesize\ttfamily]
 # Optionally define IPoIB network settings (required if planning to mount Lustre/BeeGFS over IB)
 [sms](*\#*) for ((i=0; i<$num_computes; i++)) ; do
-  wwctl node set --yes ${c_name[$i]} --netdev=ib0 --ipaddr=${c_ipoib[$i]} --netmask=${ipoib_netmask}
+  wwctl node set --yes ${c_name[$i]} --netname ib --netdev=ib0 --ipaddr=${c_ipoib[$i]} --netmask=${ipoib_netmask}
```

Comment:
This is the only place where you do not use a `=`. I don't mind, but maybe for consistency add a `=`.